lib/pthreads/pthreads.nit

```diff
@@ -236,19 +236,26 @@ end
 abstract class Thread
 	super Finalizable
 
+	# Type returned by `main`
+	type E : nullable Object
+
 	private var native: nullable NativePthread = null
 
+	# Is this thread finished ? True when main returned
+	var is_done = false
```

Comment:
gitpick: the `is_done` should not be introduced in the same commit than `E`. I think that having highly focussed commits helps to read them. (note no need to rebase for that, it is juste an information for future PR)